xcube/core/store/fs/impl/dataset.py

```diff
@@ -376,13 +377,18 @@ def open_data(self,
     @classmethod
     def create_env_session(cls, fs):
         if isinstance(fs, s3fs.S3FileSystem):
-            return rasterio.env.Env(aws_secret_access_key=fs.token,
-                                    aws_access_key_id=fs.key,
-                                    aws_session_token=fs.token,
-                                    region_name=fs.client_kwargs.get(
-                                        'region_name',
-                                        'eu-central-1'
-                                    ),
+            # Create an AWSSession object as Passing abstract session keyword
```

Comment:
Please remove comment, The code is pretty clear and stands for itself.

If you insist on keeping a comment here, please rephrase as I think it's hard to understand in its current state.

Reply:
done